### ISSUE TITLE
[Accessibility]: Notepad is not "immediately" focusable

### ISSUE THREAD

**Issue by fkling** [sourcegraph/sourcegraph#35145]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34198

### Problem description

The notepad can only focused after navigation through all search results, which is inconvenient if there are many search results.

1. Enable notepad by going to https://sourcegraph.com/notebooks and click the "Enable Notepad" button (notepad should be visible in the bottom right corner).
2. Perform a search on https://sourcegraph.com/search that yields a non-trivial amount of search results.
3. Try to tab to notepad.

### Expected behavior

There should be an easier way to navigate to notepad than tabbing through all search results.

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.

**Comment by umpox** [comment]
@limitedmage Perhaps the work in https://github.com/sourcegraph/sourcegraph/pull/37577 could also help here?
